runtimes/alibi-detect/mlserver_alibi_detect/runtime.py

```diff
@@ -136,6 +141,18 @@ def _detect(self, payload: InferenceRequest) -> InferenceResponse:
 
         return self._encode_response(self._postproc_pred(pred))
 
+    def _log_metrics(self, current_pred: dict) -> None:
+        if "data" not in current_pred:
+            return
+
+        data = current_pred["data"]
+        if "is_drift" in data:
+            # NOTE: is_drift will hold an array which could be larger than 1
```

Comment:
To give context here, for univarite detectors (like `KSDrift`) `is_drift` will be an array like `'is_drift': array([1, 0])` if `predict(..., drift_type='feature')`, otherwise an int if `predict(..., drift_type='batch')`. Whereas for multivariate detectors like `MMDDrift` it will always be an int...

So above looks good, but just noting that currently the tests only test the `drift_type='feature'` behaviour...